src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -160,18 +173,66 @@
         >Return to this step to finish it</router-link>
       </section>
 
-      <section v-if="getCreateResolutionStep.validationDetail.valid"
-        class="section-container upload-success-message"
+      <section id="resolution-summary-section-3"
+               v-if="isTypeCoop && !allResolutionValidationItemsInvalid"
+               class="section-container rounded-bl-0"
+               :class="{ 'invalid-section': !getCreateResolutionStep.validationDetail.valid }"
       >
-        <v-row no-gutters>
-          <v-col md="1">
-            <v-icon class="success-chk">mdi-check</v-icon>
+        <span v-if="!getCreateResolutionStep.validationDetail.valid">
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text ml-1 mr-2">This step is unfinished.</span>
+          <router-link
+            :to="{ path: `/${RouteNames.CREATE_RESOLUTION}` }"
+          >Return to this step to finish it</router-link>
+        </span>
+        <v-row class="ml-2 mt-6" no-gutters>
+          <v-col md="3">
+            <strong>Resolution Date</strong>
           </v-col>
-          <v-col v-if="isTypeCoop" md="11" id="file-name-col">
-            <span>FILE_NAME_PLACEHOLDER</span>
+          <v-col md="9" class="ml-n1">
+            {{resolutionDate}}
```

Comment:
wrong date format

Reply:
What does this mean? Does someone have to fix this yet, or can you do it in this ticket?

Reply:
i will be fixing it.  i just noted it.  it's a note for myself.

Reply:
updated to expected date format